admin/banner_manager.php

```diff
@@ -130,7 +130,7 @@
 
           $sql = "UPDATE " . TABLE_BANNERS . "
                   SET
-                    date_scheduled = :scheduledDate,
+                    date_scheduled = DATE_ADD(:scheduledDate, INTERVAL '00:00:00' HOUR_SECOND),

```

Comment:
I do not believe so as around line 69 is:
        $expires_date = zen_db_prepare_input($_POST['expires_date']) == '' ? 'null' : zen_format_date_raw($_POST['expires_date']);
        $expires_impressions = zen_db_prepare_input($_POST['expires_impressions']);
        $date_scheduled = zen_db_prepare_input($_POST['date_scheduled']) == '' ? 'null' : zen_format_date_raw($_POST['date_scheduled']);

So far I cannot reproduce an issue. Are you seeing one?